src/components/UserProfile/CompletionWrapper.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { withTheme } from 'emotion-theming';
+import { compose } from 'recompose';
+import { injectState } from 'freactal/lib/inject';
+import { VictoryPie } from 'victory-pie';
+
+const enhance = compose(withTheme, injectState);
+
+const CompletionWrapper = ({
+  completed,
+  theme,
+  state: { loggedInUser },
```

Comment:
can this be abstracted a bit? maybe it doesn't need to be, but it feels weird for this particular component to care specifically about user state

Reply:
looks like you're not using it actually

Reply:
hmm yeah, it was there  when I had the profile image in there as well.  removed now!

any idea why eslint isn't warning on unused destructured arguments? Does atom warn you?